src/cosmwasm/wasm/v1/query.ts

```diff
@@ -1,217 +1,234 @@
 /* eslint-disable */
-import Long from "long";
-import _m0 from "protobufjs/minimal";
 import { PageRequest, PageResponse } from "../../../cosmos/base/query/v1beta1/pagination";
-import { ContractCodeHistoryEntry, ContractInfo, Model } from "./types";
-
+import { ContractInfo, ContractCodeHistoryEntry, Model } from "./types";
+import * as _m0 from "protobufjs/minimal";
+import { isSet, DeepPartial, Exact, Long, bytesFromBase64, base64FromBytes, Rpc } from "../../../helpers";
 export const protobufPackage = "cosmwasm.wasm.v1";
-
 /**
  * QueryContractInfoRequest is the request type for the Query/ContractInfo RPC
  * method
  */
+
 export interface QueryContractInfoRequest {
   /** address is the address of the contract to query */
   address: string;
 }
-
 /**
  * QueryContractInfoResponse is the response type for the Query/ContractInfo RPC
  * method
  */
+
 export interface QueryContractInfoResponse {
   /** address is the address of the contract */
   address: string;
-  contractInfo?: ContractInfo;
+  contractInfo: ContractInfo;
 }
-
 /**
  * QueryContractHistoryRequest is the request type for the Query/ContractHistory
  * RPC method
  */
+
 export interface QueryContractHistoryRequest {
   /** address is the address of the contract to query */
   address: string;
   /** pagination defines an optional pagination for the request. */
+
   pagination?: PageRequest;
 }
-
 /**
  * QueryContractHistoryResponse is the response type for the
  * Query/ContractHistory RPC method
  */
+
 export interface QueryContractHistoryResponse {
   entries: ContractCodeHistoryEntry[];
   /** pagination defines the pagination in the response. */
+
   pagination?: PageResponse;
 }
-
 /**
  * QueryContractsByCodeRequest is the request type for the Query/ContractsByCode
  * RPC method
  */
+
 export interface QueryContractsByCodeRequest {
-  /** grpc-gateway_out does not support Go style CodID */
+  /**
+   * grpc-gateway_out does not support Go style CodID
+   * pagination defines an optional pagination for the request.
```

Comment:
this comment seems to be the only one due to a newline that is stuck. I suggest we fix the proto for this one.